Sources/Jobs/JobQueueHandler.swift

```diff
@@ -27,6 +27,7 @@ final class JobQueueHandler<Queue: JobQueueDriver>: Sendable {
         self.logger = logger
         self.jobRegistry = .init()
         self.options = options
+        Meter(label: "swift.jobs.worker.count").increment(by: Double(numWorkers))
```

Comment:
When does this number decrement?

Reply:
What happens when an app crashes?

Reply:
We should not decrement this gauge. We can use it for alerting purposes because once an app crashes or shutdown, after a while prometheus will return 0

Reply:
> We should not decrement this gauge. We can use it for alerting purposes because once an app crashes or shutdown, after a while prometheus will return 0

Are you saying that prometheus will revert a Meter to zero if the connection from the app is lost due to a crash? What does that mean for all of the other Meters?

Reply:
I should have phrased the previous sentence better. Prometheus would not reset the meter to zero, there would be no data if one where to query for most recent update. 

```pql
max_over_time(swift_jobs_worker_count{}[5m]))
```

To be safe, I will try to decrement during graceful shutdown. I can see the worker count might be misleading in a continuous deployment environment.